build/requirements.txt

```diff
@@ -5,3 +5,8 @@ gspread==0.2.5
 gunicorn==19.3.0
 oauth2client==1.4.12
 requests[security]==2.7.0
+
+# requirements for airtable to google groups sync
+#requests[security]==2.7.0
```

Comment:
Not sure how to specify that the airtable sync requires `requests[security]` too. Tho separating out which dependencies apply to which app probably doesn't matter much.